helix-term/src/keymap.rs

```diff
@@ -62,188 +153,187 @@ impl DerefMut for Keymaps {
 
 impl Default for Keymaps {
     fn default() -> Keymaps {
+        use KeyTrie::*;
         let normal = hashmap!(
```

Comment:
Can we change hashmap macro to automatically add the `Leaf`?

Reply:
I made a new `keymap!` macro inspired by `mode_info!` which takes it a bit further :) https://github.com/helix-editor/helix/pull/454/files#diff-8b8f04b030b7fdea89f3d9ec2b7aeb6a33959ee4c6cb27f0d574f115b474d151R157-R161